test/Microsoft.ML.Tests/TrainerEstimators/TreeEstimators.cs

```diff
@@ -405,6 +458,82 @@ public void LightGbmMulticlassEstimatorCompareOva()
             Done();
         }
 
+        /// <summary>
+        /// LightGbmMulticlass UsingSigmoids test 
+        /// </summary>
+        [LightGBMFact]
+        public void LightGbmMulticlassEstimatorCompareOvaUsingSigmoids()
+        {
+            var sigmoid = .790;
```

Comment:
```suggestion
            var sigmoidScale = .790;
```
to match some inline comments below.

Reply:
I renamed it directly in VisualStudio so I could make sure I got all its references as well. Fixed.